test/organization_test.go

```diff
@@ -0,0 +1,18 @@
+package test
+
+func (s *CLITestSuite) TestOrganization() {
+	tests := []CLITest{
+		{args: "organization describe", fixture: "organization/describe.golden"},
+		{args: "organization describe -o json", fixture: "organization/describe-json.golden"},
+		{args: "organization describe -o yaml", fixture: "organization/describe-yaml.golden"},
```

Comment:
IMO, you only need to test one of these (verify existence of the `--output` flag and serialization tags). The other is covered by unit tests.